sa/saro.go

```diff
@@ -116,10 +117,13 @@ func (ssa *SQLStorageAuthorityRO) GetRegistration(ctx context.Context, req *sapb
 		model, err = selectRegistration(ssa.dbReadOnlyMap.WithContext(ctx), query, req.Id)
 		if err != nil {
 			if db.IsNoRows(err) {
-				ssa.lagFactorCounter.WithLabelValues("GetRegistration", "fail").Inc()
+				ssa.lagFactorCounter.WithLabelValues("GetRegistration", "notfound").Inc()
+			} else {
+				ssa.lagFactorCounter.WithLabelValues("GetRegistration", "other").Inc()
 			}
+		} else {
+			ssa.lagFactorCounter.WithLabelValues("GetRegistration", "found").Inc()
```

Comment:
It would be more idiomatic to use an early return. Then you end curly-brace becomes an implicit `else`.

```suggestion
        return nil, err
		}
		ssa.lagFactorCounter.WithLabelValues("GetRegistration", "found").Inc()
```